.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ on:
 name: CI
 
 env:
-  CI_RUST_TOOLCHAIN: 1.46.0
+  CI_RUST_TOOLCHAIN: stable
```

Comment:
let's lock to 1.50.0?

Reply:
b8b3caf Locked to 1.51 
Next stable version will release at 2021-05-06 
<https://forge.rust-lang.org/#current-release-versions>